FX/Marini_FX linker/Marini_FX linker toggle autostart.lua

```diff
@@ -0,0 +1,63 @@
+-- @noindex
+
+local header = [[
+
+
+-- >>>> Marini FX Linker
+-- This code is generated by Marini FX Linker. It should NOT be modified by the user
+-- It starts the background synching task on startup automatically
+-- If you don't want it, you can delete this block or run the "Marini_FX linker Autostart Toggle" script
+]]
+
+local body = [[
+
+reaper.Main_OnCommand(reaper.NamedCommandLookup("_RS010746d83910eb4dc5d6fbc58d2e166ca30e5180"), -1)
+
+]]
+
+local trailer = [[
+-- <<<< Marini FX Linker]]
+
+local startupFile = reaper.GetResourcePath().."/Scripts/__startup.lua"
+local read = io.open(startupFile, "r")
+local content = ""
+if read then
+  content = read:read("*all")
+  read:close()
+end
+
+local pattern = "%-%- >>>> Marini FX Linker.*%-%- <<<< Marini FX Linker"
+
+local function removeTrailingNewlines(inputString)
+    local cleanedString = inputString:gsub("[\r\n]+$", "")
+    return cleanedString
+end
+
+local function addAutoStartup()
+  local write = io.open(startupFile, "w")
+  if not write then return end
+  if content:match(pattern) then
+    print("c'è già autostart")
```

Comment:
Stray debug statement?

Reply:
Yeah, my bad. Removed is in [a4a2e09](https://github.com/ReaTeam/ReaScripts/pull/1301/commits/a4a2e09a5d9834a08b3174494fd620f2599e5c52)